composer.json

```diff
@@ -13,17 +13,17 @@
         "exclude-from-classmap": [ "Tests/" ]
     },
     "require": {
-        "propel/propel": "dev-master",
+        "propel/propel": "2.0.0-beta2",
         "symfony/console": "^2.8|^3.0|^4.0",
         "symfony/dependency-injection": "^2.8|^3.0|^4.0",
         "symfony/framework-bundle": "^2.8|^3.0|^4.0",
         "symfony/security-acl": "^2.8|^3.0|^4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.0",
+        "phpunit/phpunit": "^9.5",
         "sensio/framework-extra-bundle": "^4.0",
         "symfony/form": "^2.8|^3.0|^4.0",
-        "fzaninotto/faker": "^1.5",
-        "mikey179/vfsStream": "^1.6"
+        "fzaninotto/faker": "dev-master",
```

Comment:
❓ Any specific reason to use the `dev-master` version here and not a normal one?

Reply:
The latest version does not allow php 8 https://github.com/fzaninotto/Faker/blob/v1.9.2/composer.json#L17